app/src/androidTest/java/com/android/solvit/shared/ui/map/GetDirectionsBubbleTest.kt

```diff
@@ -0,0 +1,75 @@
+package com.android.solvit.shared.ui.map
+
+import android.content.Context
+import android.content.Intent
+import android.net.Uri
+import androidx.compose.ui.test.*
+import androidx.compose.ui.test.junit4.createComposeRule
+import androidx.test.core.app.ApplicationProvider
+import com.android.solvit.shared.model.map.Location
+import org.junit.Rule
+import org.junit.Test
+
+class GetDirectionsBubbleTest {
+
+  @get:Rule val composeTestRule = createComposeRule()
+
+  @Test
+  fun testGetDirectionsBubble_DisplaysLocationName() {
+    val location = Location(name = "Test Location", latitude = 0.0, longitude = 0.0)
+    composeTestRule.setContent { GetDirectionsBubble(location = location, onDismiss = {}) }
+
+    composeTestRule.onNodeWithText("Test Location").assertIsDisplayed()
+  }
+
+  @Test
+  fun testGetDirectionsBubble_GetDirectionsButton() {
+    val location = Location(name = "Test Location", latitude = 0.0, longitude = 0.0)
+    composeTestRule.setContent { GetDirectionsBubble(location = location, onDismiss = {}) }
+
+    composeTestRule.onNodeWithText("Get Directions").assertIsDisplayed()
+  }
+
+  @Test
+  fun testGetDirectionsBubble_ClickGetDirectionsButton() {
+    val location = Location(name = "Test Location", latitude = 0.0, longitude = 0.0)
+    composeTestRule.setContent { GetDirectionsBubble(location = location, onDismiss = {}) }
+
+    composeTestRule.onNodeWithText("Get Directions").performClick()
+
+    val expectedUri = Uri.parse("google.navigation:q=0.0,0.0")
```

Comment:
Nit: It would be a little nicer to use the values of latitude and longitude instead of typing them by hand
```suggestion
    val expectedUri = Uri.parse("google.navigation:q=${testLocation.latitude},${testLocation.longitude}")
```